app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,57 @@
+<p style="color: green"><%= notice %></p>
+
+
+  <div class="recipe-header">
+      
+    <div class="button-header">
+      <h2 class><%= @recipe.name%></h2>
+    </div>
+
+      <div class="publicR">
+      <h3 class="r-subtitle">
+      Preparation Time: <span class="r-sub-text"><%= @recipe.preparation_time %> min(s)</span>
+      </h3>
+      
+        <% if @recipe.public %>
+        <div class="checker-container">
+          <input class="check-box" type="checkbox" id="checkbox-check" checked />
+          <label class="check-label" for="checkbox-check">Public</label>
+        </div>
+
+        <% end %> <% if !@recipe.public %>
+        <div class="checker-container">
+          <input class="check-box" type="checkbox" id="checkbox-check"/>
+          <label class="check-label" for="checkbox-check">Public</label>
+        </div>
+        <% end %>
+      </div>
+
+    <h3 class="r-subtitle">
+      Cooking Time: <span class="r-sub-text"><%= @recipe.cooking_time %> min(s)</span>
+    </h3>
+
+    <div class="r-sub-text">
+      <h3>Step by step description:</h3>
+      <p class="r-sub-text"><%= @recipe.description %></p>
+    </div>
+
+  </div>
+  
+<div class ="edith-button">
+  <button class="back1"><%= link_to "Edit this recipe", edit_recipe_path(@recipe) %></button>
+  <br>
+  <button class="back1"><%= link_to "Back to recipes", recipes_path %></button>
+<br>
+  <%= button_to "Destroy this recipe", @recipe, method: :delete %>
+</div>
```

Comment:
- [ ] The Food table shown under the recipe detail page is missing. Kindly ensure that the food table is displayed under each recipe, as in can be seen in the [wireframe design](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/recipe-app/images/recipe_food.png?token=GHSAT0AAAAAABW7GI45NILK6L4GVBGZ3IZSY5Z5W7A):

**Your design:**        |  **The Original design:**
:-------------------------:|:-------------------------:
<img width="800" alt="Your desgin" src="https://user-images.githubusercontent.com/12745474/211170507-a8a24a2e-d066-4a11-acb1-fc2ebbcc9209.png"> |  <img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/211170609-e12c96e9-a789-48a0-9f44-cbc0774aac70.png">